libsol/message.c

```diff
@@ -92,6 +112,11 @@ int process_message_body(const uint8_t* message_body,
             case ProgramIdUnknown:
                 display_instruction_info[display_instruction_count++] = info;
                 break;
+            case ProgramIdComputeBudget:
+                handle_compute_budget_instruction_display(display_instruction_info,
```

Comment:
this is not the place to make this decision. it should be made near the "expert mode" resolution for other instructions